marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-job-monitor.json

```diff
@@ -0,0 +1,14 @@
+{
+  "role-name": "data-hub-job-monitor",
+  "description": "This role grants access to read job monitoring related data",
```

Comment:
Is the intent of this role to distinguish between users that can read job data for the purpose of e.g. viewing records and when they've been processed by steps, vs users that want to perform "monitoring" actions that are should not be accessible to all of the former group of users?

I ask because if so, I don't think this description really conveys that. "read job monitoring related data" is already provided by the data-hub-job-reader role.

I am guessing the description should be something like "Permits access to job monitoring features in DHF" (note that it should start with "Permits" for consistency with existing DHF roles). That conveys that - hey, just because you can read job data doesn't mean you're able to access job monitoring features; you're going to need this role if you want to do that.

Reply:
Updated description